src/styles/blocks/contacts.scss

```
@@ -0,0 +1,225 @@
+.contacts {
+  &__content {
+    gap: 60px;
+    margin-bottom: 80px;
+
+    @include onTablet {
+      display: flex;
+      margin-bottom: 60px;
+    }
+
+    @include onDasktop {
+      display: flex;
+      margin-bottom: 70px;
+    }
+  }
+  &__media {
+    display: flex;
+    justify-content: space-between;
+    margin-bottom: 40px;
+
+    @include onTablet {
+      flex-direction: column;
+      justify-content: start;
+    }
+
+    @include onDasktop {
+      flex-direction: column;
+      justify-content: start;
+    }
+  }
+  &__logo {
+
+    @include onTablet {
+      margin-bottom: 60px;
+    }
+
+    @include onDasktop {
+      margin-bottom: 60px;
+    }
+  }
+  &__social {
+
+    @include onTablet {
+      display: flex;
+    }
+
+    @include onDasktop {
+      display: flex;
+    }
+  }
+  &__fb {
```

Add blank line between the selectors everywhere for better readability
```suggestion
  }
  
  &__fb {
```